integration/testdata/custom/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+bazel build //:skaffold_example.tar
+TAR_PATH=$(bazel info bazel-bin)
+docker load -i $TAR_PATH/skaffold_example.tar
+
+
+images=$(echo $IMAGES | tr " " "\n")
+
+for image in $images
+do
+    docker tag bazel:skaffold_example $image
+
+    if $PUSH_IMAGE
```

Comment:
where is this variable set?

Reply:
it's set [here](https://github.com/GoogleContainerTools/skaffold/pull/1999/files#diff-abb7e5fea9d678808b14582869c3dd47R70) -- the envs are passed in to the cmd which runs the build script.

Reply:
yes saw that later. ! thanks